.github/workflows/module-prep-release.yml

```diff
@@ -0,0 +1,128 @@
+name: Prepare Release
+# This workflow is manually dispatched. It:
```

Comment:
Could we clarify this?

`workflow_call` will only be dispatched when another workflow initiates the call. `workflow_dispatch` is for a person manually runs the workflow.

Reply:
Cleared this comment up